hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -696,7 +696,30 @@ possible configurations would overwhelm and obscure the important.
       Enables the MemStore-Local Allocation Buffer,
       a feature which works to prevent heap fragmentation under
       heavy write loads. This can reduce the frequency of stop-the-world
-      GC pauses on large heaps.</description>
+      GC pauses on large heaps.
+    </description>
+  </property>
+  <property>
+    <name>hbase.hregion.memstore.mslab.chunksize</name>
+    <value>2097152</value>
+    <description>The maximum size of a chunk in the MemStoreLAB.</description>
```

Comment:
This is bytes?

Reply:
Yeah,  Let me address this in the description. Thanks.